client/sysinfo/src/sysinfo.rs

```diff
@@ -32,14 +33,117 @@ use std::{
 	time::{Duration, Instant},
 };
 
+/// Used to represent the unit used in the benchmarks.
```

Comment:
```suggestion
/// The unit in which the [`Throughput`] (bytes per second) is denoted. 
```
`Unit` is a bit weak as name, but its not a blocker from my side.